src/app/modules/gb-explore-module/gb-explore.component.html

```diff
@@ -65,17 +65,24 @@
     <br>
     <p-accordion class="gb-additional-param-wrapper">
       <p-accordionTab header="Additional settings" [selected]="false">
-        <p>Defaults recommended by the CLSI guidelines</p>
-        <div class="gb-additional-param"><label>Number of Bootstrap iterations</label><br>
-          <input type="number" [(ngModel)]="bootR" pInputText placeholder="Number of Bootstrap iterations" />
+        <h5>Defaults recommended by the CLSI guidelines</h5>
```

Comment:
in the end this part should be removed/hidden according to what we discussed with Matthias. Maybe showing it only if a dedicate env is set, for test purposes? (only if it's worth it)

Reply:
Now it's only showing on "Dev" mode